gradle/libs.versions.toml

```diff
@@ -136,6 +136,7 @@ jackson-dataformat-xml = { module = "com.fasterxml.jackson.dataformat:jackson-da
 jackson-jaxrs-json-provider = { module = "com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider" }
 jackson-jaxrs-xml-provider = { module = "com.fasterxml.jackson.jaxrs:jackson-jaxrs-xml-provider" }
 jacoco-maven-plugin = { module = "org.jacoco:jacoco-maven-plugin", version.ref = "jacoco" }
+jaeger-core = { module = "io.jaegertracing:jaeger-core", version = "1.8.1" }
```

Comment:
why did we need to add this for java 8 ?

Reply:
It was included via the Quarkus depenedency.